src/main/java/utils/BrowserManagement.java

```diff
@@ -0,0 +1,55 @@
+package utils;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.github.cdimascio.dotenv.Dotenv;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.opera.OperaDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class BrowserManagement {
+
+    private static WebDriver driver;
+    private static Dotenv dotenv= Dotenv.load();
+
+    private BrowserManagement (){}
```

Comment:
Just a code style comment: Be careful with the character spacing. `private BrowserManagerment() {}`

Reply:
fixed